saleor/static/scss/layouts/_cart.scss

```diff
@@ -100,4 +100,7 @@
       width: 100%;
     }
   }
+  &__clear{
+    margin-right: 10px;
```

Comment:
You could use a bootstrap' spacing utility classes instead, it's also a really bad idea to use pixels. (https://getbootstrap.com/docs/4.1/utilities/spacing/)